docs/kubernetes/user-guides/driver-install.md

```diff
@@ -20,4 +20,21 @@ $ ./deploy/setup-project.sh
 $ GCE_PD_SA_DIR=/my/safe/credentials/directory    # Directory to get the service account key
 $ GCE_PD_DRIVER_VERSION=stable                    # Driver version to deploy
 $ ./deploy/kubernetes/deploy-driver.sh
-```
\ No newline at end of file
+```
+
+## Disabling particular CSI driver services
+
+Traditionally, you run the CSI controllers with the GCE PD driver in the same Kubernetes cluster.
+Though, there may be cases where you will only want to run a subset of the available driver services (for example, one scenario is running the controllers outside of the cluster they are serving (while the GCE PD driver still runs inside the served cluster), but there might be others scenarios).
+The CSI driver consists out of these services:
+
+* The **controller** service starts the GRPC server that is used for communication between the CSI components. It is depending on the GCP service account credentials.
```

Comment:
This isn't entirely accurate - the controller is the component that serves `CreateVolume`, `DeleteVolume` etc. Although from a cursory glance at the code it does seem like the controller component also depends on the metadataservice for getting the `project` and `zones` for particular operations 

I'm starting to think that maybe your goal is to not actually be to turn off particular services - instead is it to specify a "project" and "zone" external to the cluster that the driver is running on? In that case there is no need to turn off particular services, you can add some options to provide project and zone as flags and use those values if they are provided